### DISCUSSION TITLE
How to get UserInfo?

### DISCUSSION THREAD

**Discussion by harshalsc** [iTwin/itwinjs-core#3481] [Q&A] [answered]
Previously we used to fetch user info from AccessToken from @bentley/itwin-client. And in 3.0 update, UserInfo has been removed [https://www.itwinjs.org/changehistory/3.0.0/#bentleyitwin-client](https://www.itwinjs.org/changehistory/3.0.0/#bentleyitwin-client) and AccessToken is moved to @itwin/core-bentley and it is just a token string. 
Is there any alternate way to get the user info now?

**Comment by aruniverse** [comment, marked as answer]
Hi! Please use the following iTwin Platform API, https://developer.bentley.com/apis/users/operations/me/

**Reply by harshalsc** [reply]
Thanks
